client/my-sites/plugins/plugin-action/plugin-action.jsx

```diff
@@ -38,53 +38,61 @@ module.exports = React.createClass( {
 		return null;
 	},
 
-	renderToggle: function( id ) {
-		if ( this.props.disabledInfo ) {
-			return (
-				<div className="plugin-action__disabled-info">
-					<InfoPopover
-						position="bottom left"
-						popoverName={ 'Plugin Action Disabled' + this.props.label }
-						gaEventCategory="Plugins"
-						ref="infoPopover"
-						ignoreContext={ this.refs && this.refs.disabledInfoLabel }
-						>
-						{ this.props.disabledInfo }
-					</InfoPopover>
-					{ this.renderLabel( id ) }
-				</div>
-			);
-		}
+	renderDisabledInfo: function() {
+		return (
+			<div className="plugin-action__disabled-info">
+				<InfoPopover
+					position="bottom left"
+					popoverName={ 'Plugin Action Disabled' + this.props.label }
+					gaEventCategory="Plugins"
+					ref="infoPopover"
+					ignoreContext={ this.refs && this.refs.disabledInfoLabel }
+					>
+					{ this.props.disabledInfo }
+				</InfoPopover>
+				{ this.renderLabel() }
+			</div>
+		);
+	},
+
+	renderToggle: function() {
 		return (
 			<CompactToggle
 				onChange={ this.props.action }
 				checked={ this.props.status }
 				toggling={ this.props.inProgress }
 				disabled={ this.props.disabled }
-				id={ id }
+				id={ this.props.htmlFor }
 			>
-				{ this.renderLabel( id ) }
+				{ this.renderLabel() }
 			</CompactToggle>
 		);
 	},
 
-	renderChildren: function( id ) {
+	renderChildren: function() {
 		return (
 			<div>
 				<span className="plugin-action__children">{ this.props.children }</span>
-				{ this.renderLabel( id ) }
+				{ this.renderLabel() }
 			</div>
 		);
 	},
 
+	renderInner: function() {
+		if ( this.props.disabledInfo ) {
+			return this.renderDisabledInfo();
+		}
+		return (
+		0 < React.Children.count( this.props.children )
```

Comment:
Yeah this is odd. The automating that I have did this. I moved it to a new line and it looks a bit better.